README.md

```diff
@@ -62,13 +62,15 @@ Additional configuration options supported: https://github.com/axios/axios#reque
 
 Returns a promise which resolves, if successful, to an instance of `MetadataReader`.
 
-### toPassportConfig(reader)
+### toPassportConfig(reader, options = { multipleCerts: false })
 
 Transforms metadata extracts for use in Passport strategy configuration. The following strategies are currently supported:
 
 * [passport-saml](http://npmjs.org/packages/passport-saml)
```

Comment:
While you're in here, could you fix the typo (`/packages` should be `/package`) for me?